docs/search-views/fulltext-search/analyzers/index.md

```diff
@@ -7,8 +7,6 @@ _Analyzers_ enable you to break search inputs into sets of sub-values that searc
 
 You can use the [`TOKENS()` function](../queries/search-functions/tokens.md) to tokenize phrases and turn them into strings for C8QL search queries.
 
-An analyzer processes values based on its [features](features.md).
```

Comment:
Not sure why [Analyzer features](https://www.arangodb.com/docs/3.8/analyzers.html#analyzer-features) link is removed?

Reply:
As I understand it based on the last time we chatted, the items listed in the Analyzer Features section were for custom analyzers, which we do not currently support. I hid that section until we do support them, so the link was broken due to the content being removed.